FindmallocMC.cmake

```diff
@@ -41,66 +45,48 @@
 ###############################################################################
 
 
-###############################################################################
-# Required cmake version
-###############################################################################
-
+# Required cmake version ######################################################
+#
 cmake_minimum_required(VERSION 2.8.5)
 
 
-###############################################################################
-# mallocMC
-###############################################################################
-
-# we start by assuming we found mallocMC and falsify it if some
-# dependencies are missing (or if we did not find mallocMC at all)
+# mallocMC ####################################################################
+#
 set(mallocMC_FOUND TRUE)
 
 
-###############################################################################
-# preconditions
-###############################################################################
-
-if(NOT CUDA_FOUND)
-    set(mallocMC_FOUND FALSE)
-    message(STATUS "could not find CUDA, try something like find_package(CUDA REQUIRED)")
-elseif(CUDA_VERSION VERSION_LESS "5.0")
-    set(mallocMC_FOUND FALSE)
-    message(STATUS "CUDA found, but version too low (needs 5.0 or higher)")
-endif(NOT CUDA_FOUND)
-
-if(NOT Boost_FOUND)
-    set(mallocMC_FOUND FALSE)
-    message(STATUS "could not find Boost, try something like find_package(Boost REQUIRED)")
-elseif(Boost_VERSION LESS 104800)
-    set(mallocMC_FOUND FALSE)
-    message(STATUS "Boost found, but version too low (needs 1.48 or higher)")
-endif(NOT Boost_FOUND)
+# dependencies ################################################################
+#
+find_package(CUDA 5.0 REQUIRED)
+find_package(Boost 1.48.0 REQUIRED)
 
 
-# find mallocMC installation #################################################
+# find mallocMC installation ##################################################
 #
 find_path(mallocMC_ROOT_DIR
-  NAMES include/mallocMC/mallocMC.hpp
-  PATHS ENV MALLOCMC_ROOT
-  DOC "mallocMC ROOT location"
-)
+    NAMES include/mallocMC/mallocMC.hpp
+    PATHS ENV MALLOCMC_ROOT
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

BOOST_ROOT can be env. *or* cmake? Did not know that, interesting. most modules only allow one of those.

Besides that _ROOT is a bit deprecated and for env. CMAKE_PREFIX_PATH is the more modern way.
-----BEGIN PGP SIGNATURE-----
Version: APG v1.1.1

iQJCBAEBCgAsBQJViBbDJRxBeGVsIEh1ZWJsIDxheGVsLmh1ZWJsQHBsYXNtYS5u
aW5qYT4ACgkQRy0sNk4h2UtPxQ//ava4uCKhDaU/pEnJGuwN4woSw0rF1tt+AlQd
Mlgf4scpFTuNGNX4mtt8pJM1MPyIErSKSil1SufAOjiLyU8UHLRybiSpvgKWoq8K
FObrKyH07B/JPH6BCH26YQ9+pqjhkMHEoChzzER+aAMXXhBMdpiOir14tLZ+ZzOk
EkGJkF/Vv4Hin07jVY3LoUqwx0hhmX7FbH7ii/uJHLWz2xUDcumzc3TET/bXEtsT
5E+/yiwa+yTY9mrpNnnKQ6GETVSgvCk4wsS/4jCOhfSlpncFSoKx5knUk1pZqxYm
FMpUEbcNWK3JT4V9Mw5uogfVK6/+4oL6OyYEPYAnHNueAMwwGDRSu5wyG1VHvUbP
mcWVEaUVVt4MpE7r+nC8gv+smE3lnTLyronmQGOhRP0SmN+kP6enbITJl0h2MvJ0
Y1+MBUOodMSSUj7fI1ZUVInYs8yIG37krRp4DR5Ax+v8ubuHcXL8K6s23O/Nh+3K
eqenXq3jT1R8JI8kn8JoLFnFbp/ekHxOGe7hxU4s2q6EiRyAWEmFmoR5NF6kdQ47
6fyruiugKQrVhao7uvemmUXQLr/vKZEGwNI703o5gB6Eks8+WgBweNEcfPEvSKZY
WrHYJUg/BoBXFeQ2jaVEhlfxMh3hNY4N77Ts4E9SZwvaVzRZlMnzrK8OKI0DyGor
zWNVW74=
=2ioU
-----END PGP SIGNATURE-----